### PR TITLE
Add support for with_activity_data when retrieving reactions

### PR DESCRIPTION
**Pull request by vonbv**

### CODE REVIEW

File: src/stream-net/Reactions.cs [hunk 1]
```diff
@@ -193,20 +229,51 @@ public async Task<Reaction> Get(string reactionID)
 
         public async Task<IEnumerable<Reaction>> Filter(ReactionFiltering filtering, ReactionPagination pagination)
         {
-            var urlPath = pagination.GetPath();
-            var request = this._client.BuildJWTAppRequest($"reaction/{urlPath}", HttpMethod.GET);
-            filtering.Apply(request);
+            //var urlPath = pagination.GetPath();
```
Comment (Link512) [inline src/stream-net/Reactions.cs]: these can be removed

Comment (vonbv) [reply src/stream-net/Reactions.cs]: Removed

Comment (vonbv) [reply src/stream-net/Reactions.cs]: Hello, may I ask when will this make its way to the master branch? I got the latest version from nuget and this is not part of it.

Comment (Link512) [reply src/stream-net/Reactions.cs]: i will merge it and deploy it now

**Pull request merged**